gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/uikit/internal/configureInstallXcodeGenTask.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.uikit.internal
+
+import org.gradle.api.Project
+import org.jetbrains.compose.desktop.application.internal.MacUtils
+import org.jetbrains.compose.desktop.application.internal.UnixUtils
+import org.jetbrains.compose.experimental.uikit.tasks.AbstractComposeIosTask
+
+internal const val TASK_INSTALL_XCODE_GEN = "iosInstallXcodeGen"
```

Comment:
Maybe better refer tasks by value (i.e. task instance), not by name.

Reply:
Done